### ISSUE TITLE
Title event picture - top part cropped off

### ISSUE THREAD

**Issue by Collabophotos** [nautilusdigital/collabo-issues#2]
On both the list of event screen and when clicking and in the actual event, the top part of the event picture is still cropped off at the top.  

I believe this was something I had brought up in an earlier meeting.    A possible solution would be when uploading the picture, you are given the ability to move and either zoom in or out to manipulate the image to fit the area so that at least user decides where the picture is being cropped.  

cropping the top of the picture is more times than not, the better half of the picture.  

![img_0012](https://user-images.githubusercontent.com/33112490/32037160-b8ff4d42-b9d8-11e7-8d15-1f661851553c.PNG)
![img_0011](https://user-images.githubusercontent.com/33112490/32037161-b9174f3c-b9d8-11e7-9b44-7e4ad8711a4d.PNG)

**Comment by mhousser** [comment]
Agree. Event banner upload process should have the same 'preview + crop' screen as updating your profile picture. Current functionality usually just ruins the picture.

**Comment by Collabophotos** [comment]
The cover photo picture allowance seems a bit stingy. Most pictures are
larger.  Can we not push the whole thing down to allow for a larger image?
Of course not compromising the experience on the rest of the screen. The
start and end area is rather small on default settings.  When I increased
the font and screen on pixel, the thumbnail pics did not adjust
accordingly.
